### PR TITLE
Update versions for Mender 2.7.0

### PR DESCRIPTION
**Pull request by lluiscampos**
* Update versions for Mender 2.7.0
Using:
```
./autoversion.py --update --integration-version 2.7.0 --mender-binary-delta 1.2.1 --poky-version dunfell --meta-mender-version dunfell
```

* Update compatibility table for Mender 2.7.0
* Add changelog and statistics for Mender 2.7.0

### CODE REVIEW

**Comment by lluiscampos** [comment]
(do not review yet)

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 1]
```diff
@@ -6,6 +6,1242 @@ shortcode-core:
...
+
+New changes in mender-cli since 1.6.0:
+
+* Fix: Respect the --server flag from config everywhere
+* `mender-cli --record <my-file> terminal <DEVICE-ID>` records
+  the terminal session into a local file.
+  ([MEN-4318](https://tracker.mender.io/browse/MEN-4318))
+* `mender-cli` --playback <my-file> terminal` playbacks the
+  previously recorded terminal session from a local file.
+  ([MEN-4318](https://tracker.mender.io/browse/MEN-4318))
```
Comment (kacf) [inline 202.Release-information/01.Release-notes-changelog/docs.md:1003]: This entry has wrong quoting (thanks to Github for the coloring). I'm not sure if it "spreads" to subsequent entries, but maybe fix manually?

**Review by kacf — APPROVED** [review]
Looks good, but check out one quoting issue below.

File: 202.Release-information/01.Release-notes-changelog/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 202.Release-information/01.Release-notes-changelog/docs.md:1003]: Good catch, I didn't see it. Fixing

**Pull request merged**